docs/index.md

```diff
@@ -76,6 +76,9 @@ KICS scans and detects issues in following Infrastructure as Code solutions:
 <div style="min-width:80;flex:0 0 25%;display:flex;align-items:center;justify-content:center;margin:8px">
         <img alt="Azure Blueprints" src="img/logo-azure-blueprints.png"  width="80" style="min-width:80px">&nbsp;&nbsp;&nbsp;
 </div>
+<div style="min-width:80;flex:0 0 25%;display:flex;align-items:center;justify-content:center;margin:8px">
+        <img alt="Azure Blueprints" src="img/logo-github-icon.png"  width="80" style="min-width:80px">&nbsp;&nbsp;&nbsp;
```

Comment:
Hi there. I believe the `alt` text should be `GitHub` something?

Reply:
Hi, you are totally right! thank you so much! :D

Reply:
Wow, it's approved and merged now @gabriel-cx Congrats! 🥳

Reply:
Thank you @za for the help!!